oidc-provider/simple_op/requirements.txt

```diff
@@ -4,4 +4,5 @@ Jinja2==2.7.3
 oic==0.7.6
 yubico-client==1.9.1
 pyjwkest==1.0.1
-httpie
```

Comment:
It's the 'http' command in the line above. Supposed not used yet. I can take it out for now.